core/src/Platforms/android/TokenCacheAccessor.cs

```diff
@@ -134,7 +152,7 @@ public ICollection<string> GetAllAccountsAsString()
         {
             return _accountSharedPreference.All.Values.Cast<string>().ToList();
         }
-
+        /*
```

Comment:
dead code.

Reply:
see related comments

---
In reply to: [199550077](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1108#discussion_r199550077) [](ancestors = 199550077)